packages/shipjs/src/cli.js

```diff
@@ -50,8 +50,12 @@ export async function cli(argv) {
     printHelp();
     return;
   }
-  const opts = removeDoubleDash(
-    parseArgs(argSpec, { permissive: false, argv })
-  );
-  await fn(opts);
+  try {
+    const opts = removeDoubleDash(
+      parseArgs(argSpec, { permissive: false, argv })
+    );
+    await fn(opts);
+  } catch (err) {
+  	console.log('' + err);
```

Comment:
We should rather only console.log when `error.code === 'ARG_UNKNOWN_OPTION'`, and throw in other cases I think

Reply:
@jamesgeorge007 I agree with Haroen.
Can you specifically console.log when error code is that, and otherwise just throw it as is?